### PR TITLE
TW-18 A friendly "not found" page

### PR DESCRIPTION
**Pull request by marktheunissen**
Adding the functionality to the Apache plugin to provide a default landing page in the case of a student mistyping their url:

e.g.
http://herpderpderpderpderp.course.training.com/ , Apache will trigger a 404 handler that will serve the landing page.

It will also show the landing page on:

http://course.training.com/
http://training.com/

The page will contain instructions on how to access the controller or for students, info on how to format their urls properly. It can also contain information on the course, like schedule, etc, and is fully customizable. Just commit to a Github repo and provide the URL! Done.

### CODE REVIEW

**Comment by marktheunissen** [comment]
@jaredjohn As a follow up to this, please could you make a nice sample landing page with Training Wheels branding in here: https://github.com/trainingwheels/sample-landing-page.git ? Don't spend too much time, it's just a sample.

File: playbooks/controller/mongo-reset-data.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ db.course.insert({
       "Drupal": {},
       "PHP": {
         "apc_shm_size": "89M"
-      },
+      }
```
Comment (elliotttf) [inline playbooks/controller/mongo-reset-data.js:29]: I love the smell of linted code in the morning.

**Comment by elliotttf** [comment]
This looks good to me, should we wait for @jaredjohn to work his magic before merging?

**Comment by marktheunissen** [comment]
We can merge now, that repository contains a basic page so it will be fine.

**Pull request merged**